src/com/bjorn/MakeChange/MakeChange.java

```diff
@@ -0,0 +1,45 @@
+package com.bjorn.MakeChange;
+
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * Created by bjornjohnson on 1/5/16.
+ */
+
+public class MakeChange implements IMakeChange {
+
+    private static final Map<String, Integer> CoinValueMap;
+    static
+    {
+        CoinValueMap = new LinkedHashMap<String, Integer>();
+        CoinValueMap.put("H", 50);
+        CoinValueMap.put("Q", 25);
+        CoinValueMap.put("D", 10);
+        CoinValueMap.put("N", 5);
+        CoinValueMap.put("P", 1);
+    }
+
+    public void updateCoinAmountHashMap(String cointType, HashMap<String, Integer> coinAmountHashMap) {
```

Comment:
Perils of the IDE: it has allowed this typo to cascade through the entire codebase.